src/Bookings.js

```diff
@@ -1,18 +1,19 @@
-import React from "react";
+import React, { useState } from "react";
 import Search from "./Search.js";
 import SearchResults from "./SearchResults.js";
-import FakeInfo from "./data/fakeBookings.json";
+import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
   const search = searchVal => {
     console.info("TO DO!", searchVal);
   };
+  const [bookings, setBookings] = useState(FakeBookings);
```

Comment:
same can be done also this way:
```js
const bookings = useState(FakeBookings);
```
difference is that instead of destructuring whatever pair of "state" and "setter" into an array of variables you can just simply catch only first element from useState() - the state variable.
b-coz if you catch "setter" function but don't use it - react gives that annoying warning in console that setBookings has been declared but never used.